src/rpc/mempool.cpp

```diff
@@ -1027,6 +1128,7 @@ void RegisterMempoolRPCCommands(CRPCTable& t)
         {"blockchain", &getrawmempool},
         {"blockchain", &importmempool},
         {"blockchain", &savemempool},
+        {"hidden", &getorphantxs},
```

Comment:
nit: we could test that it's hidden in the functional test, similarly to `addpeeraddress`/`getrawaddrman` (https://github.com/bitcoin/bitcoin/blob/cfb59da4b3bb34afae467691a3e901f2b5a186f3/test/functional/rpc_net.py#L338-L341)

Reply:
We seem to be limited/selective in testing for hidden RPCs in functional tests (i.e. there are more hidden RPCs than `addpeeraddress` and `getrawaddrman`).  Maybe that sort of check would/could be a larger scope than just `getorphantxs`?

I'm on the fence about testing that RPCs are hidden.  Changing from hidden to another category would be noticed in a PR review, and having a functional test for it means that more code needs to be maintained.

Reply:
> Maybe that sort of check would/could be a larger scope than just getorphantxs?

Please don't invalidate the ACKs, but personally find it useful as a reviewer to just enforce expected behavior in tests, even minor ones like this. It's also *documentation* to the future reader as well; the functional tests are often the first place I look for intention of RPCs, without having to dig into git(hub) history to discover it.